example/extending/newlang/8-python-mypy/build.mill

```diff
@@ -0,0 +1,225 @@
+// == Python MyPy Support
```

Comment:
Let's follow the typescript setup: MyPy support should be included in the first example. The last example should be for third-party dependencies and bundling